docs/index.md

```diff
@@ -67,12 +66,12 @@ The easiest way to get started with Swarm is to use the
 [official Docker image](https://registry.hub.docker.com/_/swarm/).
 
 ```bash
-docker pull swarm
+$ docker pull swarm
```

Comment:
Actually the Docs style guide suggests that's not the case:

https://docs.docker.com/project/doc-style/#code-and-ui-text-styling

Ping @fredl